Library/MeterGeometry.h

```diff
@@ -0,0 +1,68 @@
+#pragma once
+#ifndef __METERGEOMETRY_H__
+#define __METERGEOMETRY_H__
+
+#include "Meter.h"
+#include <vector>
+#include "..\Common\Gfx\Shape.h"
+
+class MeterGeometry :
+	public Meter
+{
+public:
+	MeterGeometry(Skin* skin, const WCHAR* name);
+	~MeterGeometry();
+
+	MeterGeometry(const MeterGeometry& other) = delete;
+	MeterGeometry& operator=(MeterGeometry other) = delete;
+
+	virtual UINT GetTypeID() { return TypeID<MeterGeometry>(); }
```

Comment:
`override` goes after the `()`. See e.g. [this](https://github.com/rainmeter/rainmeter/blob/22066aaf92514fc665806f7edb733267f82fe37b/Library/MeasureRecycleManager.h#L22).